javafmllanguage/src/main/java/net/minecraftforge/fml/javafmlmod/FMLModContainer.java

```diff
@@ -43,11 +45,14 @@ public FMLModContainer(IModInfo info, String className, ModFileScanData modFileS
         final FMLJavaModLoadingContext contextExtension = new FMLJavaModLoadingContext(this);
         this.contextExtension = () -> contextExtension;
 
-
         try {
             var moduleName = info.getOwningFile().moduleName();
             var module = gameLayer.findModule(moduleName)
                 .orElseThrow(() -> new IllegalStateException("Failed to find " + moduleName + " in " + gameLayer));
+
+            Class<?> fmlConstructEvent = Class.forName( "net.minecraftforge.fml.event.lifecycle.FMLConstructModEvent", true, Thread.currentThread().getContextClassLoader());
+            this.contextObject = (Event) fmlConstructEvent.getConstructor(ModContainer.class, ModLoadingStage.class).newInstance(this, getCurrentState());
```

Comment:
`getCurrentState()` is always `ModLoadingStage.CONSTRUCT` here, so can inline the constant
```suggestion
            this.contextObject = (Event) fmlConstructEvent.getConstructor(ModContainer.class, ModLoadingStage.class).newInstance(this, ModLoadingStage.CONSTRUCT);
```